mcs/class/monodoc/Resources/mdoc-html-utils.xsl

```diff
@@ -2632,52 +2625,16 @@ SkipGenericArgument: invalid type substring '<xsl:value-of select="$s" />'
 			</xsl:call-template>
 		</xsl:variable>
 
-		<xsl:variable name="gen-type">
-			<xsl:call-template name="GetEscapedParameter">
-				<xsl:with-param name="orig-parameter-type" select="$ptype" />
-				<xsl:with-param name="parameter-type">
-					<xsl:variable name="nested">
-						<xsl:call-template name="GetEscapedParameter">
-							<xsl:with-param name="orig-parameter-type" select="$ptype" />
-							<xsl:with-param name="parameter-type" select="$ptype" />
-							<xsl:with-param name="parameter-types" select="$type/Docs/typeparam" />
-							<xsl:with-param name="escape" select="'`'" />
-							<xsl:with-param name="index" select="1" />
-						</xsl:call-template>
-					</xsl:variable>
-					<xsl:choose>
-						<xsl:when test="$nested != ''">
-							<xsl:value-of select="$nested" />
-						</xsl:when>
-						<xsl:otherwise>
-							<xsl:value-of select="$ptype" />
-						</xsl:otherwise>
-					</xsl:choose>
-				</xsl:with-param>
-				<xsl:with-param name="parameter-types" select="$member/Docs/typeparam" />
-				<xsl:with-param name="escape" select="'``'" />
-				<xsl:with-param name="index" select="1" />
-			</xsl:call-template>
-		</xsl:variable>
-
 		<!-- the actual parameter type -->
 		<xsl:variable name="parameter-type">
-			<xsl:choose>
-				<xsl:when test="$gen-type != ''">
-					<xsl:value-of select="$gen-type" />
-					<xsl:value-of select="$pmodifier" />
-				</xsl:when>
-				<xsl:otherwise>
-					<xsl:value-of select="concat($ptype, $pmodifier)" />
-				</xsl:otherwise>
-			</xsl:choose>
+			<xsl:value-of select="concat($ptype, $pmodifier)" />
 		</xsl:variable>
 
 		<!-- s/</{/g; s/>/}/g; so that less escaping is needed. -->
 		<xsl:call-template name="Replace">
 			<xsl:with-param name="s">
 				<xsl:call-template name="Replace">
-					<xsl:with-param name="s" select="translate ($parameter-type, '+', '.')" />
```

Comment:
This translate is to support nested types, e.g. when given `Foo<System.Collections.Generic.List<int>.Enumerator>`. `List<int>.Enumerator` shows up as `List<int>+Enumerator` in Reflection and various other places.